src/main/java/mate/academy/spring/Main.java

```diff
@@ -1,4 +1,25 @@
 package mate.academy.spring;
 
+import mate.academy.spring.config.AppConfig;
+import mate.academy.spring.exception.UserNotSavedException;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
 public class Main {
+    public static void main(String[] args) {
+        User user1 = new User("Maksym", 21);
+        User user2 = new User("Ivan", 33);
+
+        AnnotationConfigApplicationContext context =
+                new AnnotationConfigApplicationContext(AppConfig.class);
+        UserService service = context.getBean(UserService.class);
+        try {
+            service.add(user1);
+            service.add(user2);
+        } catch (UserNotSavedException e) {
+            e.printStackTrace();
```

Comment:
`e.printStackTrace();` is bad practice
if you want to show stacktrace you should use RuntimeExceptions